libepoxy will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libepoxy it will fail in the following way:

```
2024/11/12 23:11:52 INFO running step "test/pkgconf"
2024/11/12 23:11:52 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:11:52 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:11:52 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:11:52 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:11:52 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + basename /home/build/melange-out/libepoxy-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + dev_pkg=libepoxy-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + apk info -L libepoxy-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + grep -q ^Name: usr/lib/pkgconfig/epoxy.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + basename usr/lib/pkgconfig/epoxy.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + lib_name=epoxy uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + echo usr/lib/pkgconfig/epoxy.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + grep -q '^usr/lib/pkgconfig/epoxy.pc$\|^usr/share/pkgconfig/epoxy.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + pkgconf --exists epoxy uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + grep -q ^Version: usr/lib/pkgconfig/epoxy.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + pkgconf --modversion epoxy uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 INFO 1.5.10 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + grep -q ^Libs: usr/lib/pkgconfig/epoxy.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + pkgconf --libs epoxy uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 INFO -lepoxy uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + grep -q ^Cflags: usr/lib/pkgconfig/epoxy.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN + pkgconf --cflags epoxy uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN Package x11 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN Perhaps you should add the directory containing `x11.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 WARN Package 'x11', required by 'epoxy', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:52 INFO pod ff8cc61828dde8558e3ea1806f394b8016ede1c04257b570918d0ba5499fd2e7 terminated
2024/11/12 23:11:52 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:11:52 INFO   workspace dir: /home/user/tmp/melange-workspace-371260556
2024/11/12 23:11:52 INFO   guest dir: /home/user/tmp/melange-guest-1725122555
2024/11/12 23:11:52 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libepoxy] Error 1
```

--- a/libepoxy.yaml
+++ b/libepoxy.yaml
@@ -2,7 +2,7 @@
 package:
   name: libepoxy
   version: 1.5.10
-  epoch: 3
+  epoch: 4
   description: Direct Rendering Manager runtime library
   copyright:
     - license: MIT
@@ -46,6 +46,12 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: libepoxy dev
+    dependencies:
+      runtime:
+        - libx11-dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

